pytests/tests/test_rpm_cmdline.py

```diff
@@ -176,3 +176,13 @@ def test_reinstall_as_file(utils):
     assert utils.check_package(pkgname)
     assert "Nothing to do" not in "\n".join(ret['stderr'])
     assert "Reinstalling" in "\n".join(ret['stdout'])
+
+
+# test something like "tdnf install /path/to/pkg.rpm"
```

Comment:
Add a negative test as well, where installation fails when option not speicified.